lib/paypal_express/models/response.rb

```diff
@@ -54,7 +54,8 @@ def self.from_response(api_call, kb_account_id, kb_payment_id, kb_payment_transa
                   :payment_info_shiphandleamount          => (extract(response, 'PaymentInfo', 'ShipHandleAmount') || extract(response, 'PaymentTransactionDetails', 'PaymentInfo', 'ShipHandleAmount')),
                   :payment_info_shipdiscount              => (extract(response, 'PaymentInfo', 'ShipDiscount') || extract(response, 'PaymentTransactionDetails', 'PaymentInfo', 'ShipDiscount')),
                   :payment_info_insuranceamount           => (extract(response, 'PaymentInfo', 'InsuranceAmount') || extract(response, 'PaymentTransactionDetails', 'PaymentInfo', 'InsuranceAmount')),
-                  :payment_info_subject                   => (extract(response, 'PaymentInfo', 'Subject') || extract(response, 'PaymentTransactionDetails', 'PaymentInfo', 'Subject'))
+                  :payment_info_subject                   => (extract(response, 'PaymentInfo', 'Subject') || extract(response, 'PaymentTransactionDetails', 'PaymentInfo', 'Subject')),
+                  :error_code                             => (extract(response, 'Errors', 'ErrorCode'))
```

Comment:
Can it be shared with the [reason_code](https://github.com/alenad/killbill-paypal-express-plugin/blob/acdc389d93abfa80caf6b808122e2015fc7e9ecb/lib/paypal_express/models/response.rb#L50), so that is requires no db changes?